apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-steps.js

```diff
@@ -50,7 +50,10 @@ function getTourSteps( localeSlug, referencePositioning ) {
 				animation: null,
 			},
 			options: {
-				classNames: [ 'is-with-extra-padding', 'wpcom-editor-welcome-tour__step' ],
+				classNames: {
+					desktop: null,
```

Comment:
Shouldn't this still contain `wpcom-editor-welcome-tour__step `? Same for all the other nulls.

Also, a little suspect that we have a description above for desktop only and classes here for mobile only?

Reply:
> a little suspect that we have a description above for desktop only

That's because we have, in `tour-card`:
`const description = descriptions[ isMobile() ? 'mobile' : 'desktop' ] ?? descriptions.desktop;`
So desktop is the default value for the description. We could remove `mobile: null` when desktop and mobile descriptions are the same.

> Shouldn't this still contain wpcom-editor-welcome-tour__step ? Same for all the other nulls

Yes, I didn't add that because we were using it with `is-with-extra-padding`. In this case, should we add the CSS class to every step, both for desktop and mobile? 🤔

Reply:
> That's because we have, in `tour-card`: `const description = descriptions[ isMobile() ? 'mobile' : 'desktop' ] ?? descriptions.desktop;` So desktop is the default value for the description. We could remove `mobile: null` when desktop and mobile descriptions are the same.

Yeah, this is suspect, and something that needs cleanup (elsewhere though).

> Yes, I didn't add that because we were using it with `is-with-extra-padding`, and that wasn't needed here. In this case, should we add the CSS class to every step, both for desktop and mobile, even in steps that don't have `options: { classNames: {...` at the moment? 🤔

I think we should add to all (it also brings up the issue with the approach to its full character - that a generic `..._step` class needs to be repeated for both mobile/desktop). The alternative could be to rename `is-with-extra-padding` to something a little more specific (`is-step-with-extra-padding` maybe), but since this is due to be removed next, I think let's not.

Reply:
Given the impending refactor, I also think "let's not". I'll add the generic step class to every step 👍 
Obviously not the better solution, but I have hope for the "variant refactor"